### DISCUSSION TITLE
add apache/age extension support Graph database

### DISCUSSION THREAD

**Discussion by holmofy** [supabase/supabase#13263] [Feature Requests] [answered]
https://github.com/apache/age

**Comment by ashwan-smart** [comment]
This would be great extension to get added to Supabase.

**Comment by johannbuscail** [comment]
Any update on this ?

**Comment by StGeass** [comment]
It would obviate the need to have your own DB for graph

**Comment by heikogoll** [comment, marked as answer]
So, let's upvote https://github.com/orgs/supabase/discussions/679#discussioncomment-7323662

**Reply by inian** [reply]
Please upvote in the linked discussion so that we can prioritise among all extension requests.
